radiation/python/radlw/stencils_gt4py.py

```diff
@@ -0,0 +1,4975 @@
+from gt4py.gtscript import (
+    stencil,
+    computation,
+    interval,
+    PARALLEL,
+    FORWARD,
+    exp,
+    log,
+    mod,
+)
+import sys
+
+# sys.path.insert(0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python")
+sys.path.insert(0, "..")
+from phys_const import con_amw, con_amd, con_g, con_avgd, con_amo3
+from radlw.radlw_param import (
+    nbands,
+    nplnk,
+    nrates,
+    eps,
+    ngptlw,
+    abssnow0,
+    absrain,
+    cldmin,
+    nspa,
+    nspb,
+    ngb,
+    ng01,
+    ng02,
+    ng03,
+    ng04,
+    ng05,
+    ng06,
+    ng07,
+    ng08,
+    ng09,
+    ng10,
+    ng11,
+    ng12,
+    ng13,
+    ng14,
+    ng15,
+    ng16,
+    ns02,
+    ns03,
+    ns04,
+    ns05,
+    ns06,
+    ns07,
+    ns08,
+    ns09,
+    ns10,
+    ns11,
+    ns12,
+    ns13,
+    ns14,
+    ns15,
+    ns16,
+    oneminus,
+    bpade,
+    fluxfac,
+    heatfac,
+    ntbl,
+    wtdiff,
+)
+from radphysparam import ilwcice, ilwcliq
+from config import *
+
+rebuild = False
+validate = True
+backend = "gtc:gt:cpu_ifirst"
+
+amdw = con_amd / con_amw
+amdo3 = con_amd / con_amo3
+
+
+@stencil(
+    backend=backend,
+    rebuild=rebuild,
+    verbose=True,
```

Comment:
do we want to leave verbose=True or was this for debugging?

Reply:
Was for debugging, will remove